app/javascript/components/App.js

```diff
@@ -1,17 +1,34 @@
-import React from "react";
-import PropTypes from "prop-types";
-import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
-import Header from "./Components/Header";
+import React, { Component } from 'react'
+import Header from './components/Header'
+import Footer from './components/Footer'
+import Home from './pages/Home'
+import LocationIndex from './pages/Locations/LocationIndex'
+import LocationShow from './pages/Locations/LocationShow'
+import LocationNew from './pages/Locations/LocationNew'
+import LocationEdit from './pages/Locations/LocationEdit'
+import NotFound from './pages/NotFound'
+import {
```

Comment:
Move third party pkgs/deps to the top. And, components towards the bottom. Bonus points if Alphabetized.